### DISCUSSION TITLE
Error when sign in git.uhz.de

### DISCUSSION THREAD

**Discussion by alp-h** [community/community#24035] [Other Feature Feedback, Questions, & Ideas]
<p>I tried to log in to <a href="http://git.uhz.de" rel="noopener nofollow ugc">git.uhz.de</a> with my github account but I got the error below,</p>
<p>Your account has been blocked. Please contact your GitLab administrator if you think this is an error.</p>
<p>How can I  log in?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Do what the message says and contact your admin. The message and URL combined suggest it’s a private GitLab (not GitHub!) instance, so only someone with admin rights for that system will be able to help you.</p>

**Comment by lee-dohm** [comment]
<p>Hello <a class="mention" href="/alp-h">@alp-h</a> and welcome to the GitHub Support Community.</p>
<p>As <a class="mention" href="/airtower-luna">@airtower-luna</a> pointed out, it sounds like you’re attempting to get support for Git<strong>Lab</strong>, not Git<strong>Hub</strong>. If you want support for GitLab, they have <a href="https://about.gitlab.com/support/">their own support site</a> which should be able to help you better :grinning:</p>
<p>I hope this gets solved for you soon!</p>

**Comment by alp-h** [comment]
<p>I’m sorry, my bad :frowning:</p>
